gm4_balloon_animals/data/gm4_balloon_animals/functions/debug/clear_purchase_search_data.mcfunction

```diff
@@ -0,0 +1,9 @@
+
+
+scoreboard players reset $trade_index gm4_balloon_animals_data
+scoreboard players reset $trade_applicable gm4_balloon_animals_data
+scoreboard players reset $trade_found gm4_balloon_animals_data
+scoreboard players reset $trade_count gm4_balloon_animals_data
+scoreboard players reset $uses gm4_balloon_animals_data
+scoreboard players reset $maxUses gm4_balloon_animals_data
+scoreboard players reset $incorrect_animal gm4_balloon_animals_data
```

Comment:
Please remove the debug directory from the PR.

Reply:
I have been thinking we need more debug functions in our code, especially noticeable with OoA.  It is not possible to do simple testing since you can only summon essences and an empty Orb, but not Shards or an Orb with pneuma already on them.

CE currently has a debug folder you can use to apply modifiers, which has been shown to be helpful already.

Reply:
We should have a team-internal discussion about debug folders then. If we want them, we need to have a convention and list them in our contributing guidelines.

I don't think they should be included in releases.

Reply:
They could be filtered out of releases, like tests?  But they should definitely exist in our source for development purposes